src/Microsoft.Extensions.Configuration.AzureAppConfiguration/Extensions/TimeSpanExtensions.cs

```diff
@@ -7,30 +7,48 @@ namespace Microsoft.Extensions.Configuration.AzureAppConfiguration.Extensions
 {
     internal static class TimeSpanExtensions
     {
+        private const int MaxAttempts = 63;
+
         /// <summary>
-        /// This method calculates a random exponential backoff which lies between <see cref="RefreshConstants.DefaultMinBackoff"/> and <see cref="RefreshConstants.DefaultMaxBackoff"/>.
+        /// This method calculates randomized exponential backoff which lies between <paramref name="min"/> and <paramref name="max"/>.
```

Comment:
the summary isn't correct. It's more like the method helps calculate retry backoff times for operations that occur periodically on a given interval

The interval parameter description looks like it could use some attention as well.

Reply:
Updated.